linkerd/app/outbound/src/tcp/tests.rs

```diff
@@ -697,11 +697,13 @@ async fn profile_endpoint_propagates_conn_errors() {
     // Build the outbound server
     let mut server = build_server(cfg, profiles, resolver, connect);
 
-    let svc = server.new_service(listen::Addrs::new(
-        Local(ServerAddr(([127, 0, 0, 1], 4140).into())),
-        Remote(ClientAddr(([127, 0, 0, 1], 666).into())),
-        OrigDstAddr(ep1),
-    ));
+    let svc = server.new_service(orig_dst::Addrs {
+        orig_dst: OrigDstAddr(ep1),
+        inner: listen::Addrs {
+            server: Local(ServerAddr(([127, 0, 0, 1], 4140).into())),
+            client: Remote(ClientAddr(([127, 0, 0, 1], 666).into())),
+        },
+    });
```

Comment:
nit/TIOLI: maybe there should be an `orig_dst::Addrs::new(OrigDstAddr, Local<ServerAddr>, Remote<ClientAddr>)` constructor to make this slightly less wordy? not a big deal.